src/MPO.jl

```diff
@@ -0,0 +1,141 @@
+using Random
+using Graphs
+using BijectiveDicts: BijectiveIdDict
+
+abstract type AbstractMPO <: AbstractAnsatz end
+
+"""
+    MPO <: AbstractAnsatz
+
+A Matrix Product Operator (MPO) [`Ansatz`](@ref) Tensor Network.
+"""
+struct MPO <: AbstractAnsatz
+    tn::Ansatz
+    form::Form
```

Comment:
que es form?

Reply:
it's the field containing the canonical form trait

imagine it as the field where you store the type of canonical form you're in and extra information about it (e.g. the orthogonality center in the case of the `MixedCanonical` form)

Reply:
ajá but why in MPO digo

Reply:
por lo mismo, puedes usar los mismo canonical forms en MPO